packit/source_git.py

```diff
@@ -26,11 +26,16 @@
 logger = logging.getLogger(__name__)
 
 
+# TODO: we'll need to distinct b/w 8 and 9 - 9 will follow Fedora
 class CentOSDistGit(DistGit):
-    """ Fedora and CentOS dist-git differ """
+    """ CentOS dist-git layout implementation """
 
-    # we store downstream content in source-git in this subdir
-    source_git_downstream_suffix = "SPECS"
+    # spec files are stored in this dir in dist-git
+    spec_dir_name = "SPECS"
+
+    # sources are stored in this dir in dist-git
+    # this applies to CentOS Stream 8 and CentOS Linux 7 and 8
+    source_dir_name = "SOURCES"
```

Comment:
Wouldn't be easier to just override `get_absolute_specfile_path()`?

Reply:
The main reason I implemented it this was that it felt easier to me to keep the distinction via constants with a description and have unified methods. In the end it doesn't matter to me what the exact implementation is, as long as it works.